lib/compiler-cranelift/src/config.rs

```diff
@@ -22,11 +22,11 @@ pub enum OptLevel {
     SpeedAndSize,
 }
 
-/// Global configuration options used to create an [`Engine`] and customize its
-/// behavior.
+/// Global configuration options used to create an
+/// `wasmer_engine::Engine` and customize its behavior.
 ///
-/// This structure exposed a builder-like interface and is primarily consumed by
-/// [`Engine::new()`]
+/// This structure exposed a builder-like interface and is primarily
```

Comment:
why are we using a past tense here? 

```suggestion
/// This structure exposes a builder-like interface and is primarily
```